README.md

```diff
@@ -4,11 +4,11 @@ HttpKernel adapter for use of Symfony and Laravel frameworks with PHP-PM. See ht
 
 ### Setup
 
-  1. Install PHP-PM
+  1. Install PHP-PM (is not mandatory)
 
         composer require php-pm/php-pm:dev-master
 
-  2. Install HttpKernel Adapter
+  1. Install HttpKernel Adapter
```

Comment:
Could you undo the changes to this file? Without php-pm this a bit pointless?

Reply:
The component php-pm is in the composer.json require and will be reloaded.

Reply:
Then remove the ordered list?

Reply:
I do not understand exactly

You mean so:

### Setup

  Install PHP-PM (is not mandatory)

        composer require php-pm/php-pm:dev-master

  Install HttpKernel Adapter

        composer require php-pm/httpkernel-adapter:dev-master

Reply:
For example, yes. In any case an ordered list with only one entry (1.) does not make much sense :)

Reply:
Ok I will add 1 and 2 to the list :D